.ci/jobs/cloudbeat-mbp.yml

```diff
@@ -0,0 +1,55 @@
+---
+- job:
+    name: cloudbeat/cloudbeat-mbp
+    display-name: CloudBeat-MBP
+    description: Build & package cloudbeat
+    project-type: multibranch
+    concurrent: true
+    script-path: Jenkinsfile
+    scm:
+    - github:
+        branch-discovery: no-pr
+        discover-pr-forks-strategy: merge-current
+        discover-pr-forks-trust: permission
+        discover-pr-origin: merge-current
+        discover-tags: true
+        head-filter-regex: '(main|6\.8|7\.1[6789]|8\.\d+|PR-.*|v[0-9].*)'
```

Comment:
we can remove irrelevant versions from here

Reply:
Done